src/main/java/redis/clients/jedis/params/GeoRadiusParam.java

```diff
@@ -91,10 +92,8 @@ public void addParams(CommandArguments args) {
       }
     }
 
-    if (asc) {
```

Comment:
Exactly this... Maybe just an ASC/DESC that can be used everywhere in Jedis?

Reply:
@bsbodden Check added line 95-96.